src/main/java/github/weichware10/util/data/DataPoint.java

```diff
@@ -2,6 +2,8 @@
 
 import java.util.Arrays;
 
+import github.weichware10.util.Logger;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'github.weichware10.util.Logger' import. Should be before 'java.util.Arrays'.